Dockerfile.bench

```diff
@@ -0,0 +1,51 @@
+FROM debian:buster-slim
+
+# Prerequisites of Scala Native
+RUN set -x \
+  && apt-get update \
+  && apt-get install -y clang libre2-dev libunwind-dev \
+  && rm -rf /var/lib/apt/lists/*
+
+
+RUN set -x \
+  `# workaround for error in debian-slim during openjdk installation` \
+  && mkdir -p /usr/share/man/man1 \
+  && apt-get update \
+  && apt-get install --no-install-recommends -y curl git gnupg openjdk-11-jre-headless python3 \
+  && rm -rf /var/lib/apt/lists/*
+
+RUN set -x \
+  && echo "deb https://dl.bintray.com/sbt/debian /" | tee -a /etc/apt/sources.list.d/sbt.list \
+  && curl -fsL "https://keyserver.ubuntu.com/pks/lookup?op=get&search=0x2EE0EA64E40A89B84B2DF73499E82A75642AC823" | apt-key add \
+  && apt-get update \
+  && apt-get install -y sbt \
+  && rm -rf /var/lib/apt/lists/*
+
+RUN set -x \
+ && apt-get update \
+ && apt-get install --no-install-recommends -y build-essential python3-dev python3-pip \
+ && rm -rf /var/lib/apt/lists/*
+# https://stackoverflow.com/questions/52904639/dockerfile-with-pip-grpcio-is-very-slow-to-build => fix not applied yet
+
+RUN set -x \
+  && apt-get update \
+  && apt-get install -y openjdk-11-jdk-headless \
+  && rm -rf /var/lib/apt/lists/*
+
+RUN set -x \
+  && apt-get update \
+  && apt-get install -y apt-file \
+  && apt-file update \
+  && apt-get install -y vim \
+  && apt-get install -y wget
+
+RUN set -x \
+ && wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh -O ~/miniconda.sh \
+ && bash ~/miniconda.sh -b -p ~/miniconda \
+ && . ~/miniconda/bin/activate \
+ && conda install --yes --quiet python=3.7 \
+ && conda install --yes --quiet tensorflow=2.2.0 \
+ && conda install --yes --quiet numpy=1.18.5
+
+RUN set -x \
+ && echo ". ~/miniconda/bin/activate" >> /etc/bash.bashrc
```

Comment:
Pls add a `CMD` that would run example and the other from a mounted volume (or how did you actually use this Dockerfile yourself?)

Reply:
Well, this was my dev docker container :D 
And I though we could ship it like that, and write in readme something along the lines of:
> To quickly start using this library, a development container image has been provided, with all necessary configuration.

Reply:
Oh ok but still, how did you use it? As in, wired up the container to VSCode? if so, that what does VSCode do specifically... does it mount a volume shared b/w host and container?

Reply:
https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.vscode-remote-extensionpack
It mounts working directory so it can be edited in contanier as well as outside.
You just put this in `.devcontainer` with the `devcontainer.json` - not that important config file. click `Open in container` and it works. 
> It's like magic

Reply:
Okay... anyway, would be useful to add a sample `docker run -v $(pwd):...` command or a docker-compose.yml so that this Docker image can be useful for non-VSCode users :)